Docker-Swarm-deployment/single-node/elk/database-stack.yml

```diff
@@ -29,6 +29,12 @@ services:
       - xpack.security.enabled=true
```

Comment:
upgrade to elastic, logstash, kibana to  7.12.1 . for elastic and logstash use this image ghcr.io/datakaveri/elasticsearch:7.12.1.  ghcr.io/datakaveri/logstash:7.12.1. 

the above elastic image contains s3 plugin, no need to iclude the volume. the launcher script and  https://github.com/datakaveri/iudx-deployment/blob/master/K8s-deployment/resource-server/database/elasticsearch/docker/Dockerfile

Reply:
@THARAK-RAM1